packages/elements/src/components/ui/value-table/ValueTable.stories.tsx

```diff
@@ -64,33 +64,33 @@ export const AlignValueRight: Story<ValueTableProps> = (props) => (
     <ValueTableItem
       label={"E-mail"}
       value={"user@example.com"}
-      alignValue={"flex-end"}
+      justifyContentValue={"flex-end"}
```

Comment:
Is this rename needed? It exposes implementation (which the value already does, but no need to expose it even more), and it is a breaking change. Can we skip it?

Reply:
Reverted!